src/Redux/configureStore.js

```diff
@@ -0,0 +1,26 @@
+import { combineReducers, configureStore, applyMiddleware } from '@reduxjs/toolkit';
+import { persistStore, persistReducer } from 'redux-persist';
+import storage from 'redux-persist/lib/storage';
+import coinsReducer from './coins/coins';
+import detailsReducer from './details/details';
+
+const persistConfig = {
+  key: 'main-root',
+  storage,
+};
+
+const rootReducer = combineReducers({
+  coins: coinsReducer,
+  details: detailsReducer,
+});
+
+const persistedReducer = persistReducer(persistConfig, rootReducer);
+
+const store = configureStore({
+  reducer: persistedReducer,
+}, applyMiddleware);
```

Comment:
- [ ] When I run the tests, this section produces an error. Kindly work on it

![image](https://user-images.githubusercontent.com/50941074/194429940-350617c5-ffc7-4a44-975a-f936c47ece9e.png)